### DISCUSSION TITLE
Github Issue page doesnt load

### DISCUSSION THREAD

**Discussion by prashil-g** [community/community#35365] [Other Feature Feedback, Questions, & Ideas]
Unable to open https://github.com/FasterXML/jackson-databind/issues/3328
Please find below error snapshot from chrome

![image](https://user-images.githubusercontent.com/95269280/194414098-17fd2da8-1af1-4c31-8a1f-688b9233c802.png)

**Comment by Kve-sudo** [comment]
Can someone tell me what to do after entering a command because I have no
idea what to do after typing the command. I thought that typing a command
would open a link or a file.  you write and then what happens remains the
same pure doodles I see only.

**Comment by 3kh0** [comment]
This happens every so often, here are a few quick fixes that sometimes help.

- Reload the page (easiest in the book)
- Reload and clear cache, in chrome do ctrl+shift+r
- Disconnect from wifi, and reconnect
- Open cmd and do ipconfig/release and then ipconfig/renew
- Turn off your computer, wait 10 seconds, turn it back on.
- Try a different browser
- Clear cookies and site data from github.com
- Reboot your router
These are some quick and easy fixes, this issue happens from time to time again, and these fixes seems to fix it for me.

If this helped please mark this as the answer! ❤️

**Comment by prashil-g** [comment]
I tired the steps you mentioned @3kh0 but it didn't help. Just out of curiosity, can you access the link shared?
https://github.com/FasterXML/jackson-databind/issues/3328

**Reply by 3kh0** [reply]
No I can not

**Comment by daeho-ro** [comment]
Since it is marked as CVE, maintainer blocked the public access I guess.

<img width="1246" alt="image" src="https://user-images.githubusercontent.com/40587651/194470838-e0ca230a-da2e-4c04-880d-491bacc4c9f2.png">

**Reply by prashil-g** [reply]
> Since it is marked as CVE, maintainer blocked the public access I guess.
> 
> <img alt="image" width="1246" src="https://user-images.githubusercontent.com/40587651/194470838-e0ca230a-da2e-4c04-880d-491bacc4c9f2.png">

If access is denied, it should show different error page I think?

**Reply by daeho-ro** [reply]
That's true.

**Reply by 3kh0** [reply]
That might be the case, someone had sent me a link to a issue that I was not allowed to view and I had gotten the unicorn.
